### PR TITLE
enh(ui): Remove Switcher and replace it with @material-ui switch in Extensions manager page

### PR DESCRIPTION
**Pull request by CoralieRubin**
I deleted two files from centreon-ui Swtich and Switcher and replace them for Switch material-ui.
I adapted some files so that the logic continues to work and replace the word switcher by switch.

### CODE REVIEW

File: src/TopFilters/index.js [hunk 1]
```diff
@@ -25,7 +27,7 @@ class TopFilters extends Component {
               {fullText ? (
                 <div
                   className={clsx(
-                    styles['container__col-md-3'],
+                    styles['container__co"top"l-md-3'],
```
Comment (kduret) [inline src/TopFilters/index.js]:
```suggestion
                    styles['container__col-md-3'],
```

**Comment by kduret** [comment]
filters' layout seems not good
image

File: src/TopFilters/index.js (hunk 1 above)
Comment (CoralieRubin) [reply src/TopFilters/index.js]: modification made !

File: src/TopFilters/index.stories.jsx [hunk 2]
```diff
@@ -16,15 +16,15 @@ export const normal = () => (
         console.log(a);
       },
     }}
-    switchers={[
+    switchs={[
```
Comment (bdauria) [inline src/TopFilters/index.stories.jsx]:
```suggestion
    switches={[
```

File: src/TopFilters/index.js [hunk 3]
```diff
@@ -40,41 +42,53 @@ class TopFilters extends Component {
...
 
               <div className={clsx(styles.container__row)}>
-                {switchers
-                  ? switchers.map((switcherColumn, index) => (
-                    <div
-                      key={`switcherSubColumn${index}`}
-                      className={filterStyles['switch-wrapper']}
-                    >
-                      {switcherColumn.map(
+                {switchs
```
Comment (bdauria) [inline src/TopFilters/index.js]:
```suggestion
                {switches
```

File: src/TopFilters/index.js [hunk 4]
```diff
@@ -40,41 +42,53 @@ class TopFilters extends Component {
...
+                  ? switchs.map((switchColumn, index) => (
```
Comment (bdauria) [inline src/TopFilters/index.js]:
```suggestion
                  ? switches.map((switchColumn, index) => (
```

**Comment by kduret** [comment]
Search filter and switches should be on the same line
image

File: src/TopFilters/index.js [hunk 5]
```diff
@@ -6,117 +6,122 @@
...
 
-class TopFilters extends Component {
-  render() {
-    const { fullText, switches, onChange } = this.props;
+const useStyles = makeStyles({
+  labelFontSize: {
+    fontSize: '13px',
+  },
+});
+const TopFilters = (props) => {
```
Comment (Thebarda) [inline src/TopFilters/index.js]: You can spread props like this: 
```suggestion
const TopFilters = ({ fullText, switches, onChange }) => {
```

File: src/TopFilters/index.js [hunk 6]
```diff
@@ -1,108 +1,126 @@
...
 /* eslint-disable react/no-array-index-key */
-/* eslint-disable prettier/prettier */
 /* eslint-disable react/jsx-filename-extension */
 /* eslint-disable react/prop-types */
 /* eslint-disable react/prefer-stateless-function */
 
-import React, { Component } from 'react';
+import React from 'react';
 import clsx from 'clsx';
+import { Switch, FormControlLabel, makeStyles } from '@material-ui/core';
```
Comment (bdauria) [inline src/TopFilters/index.js]:
```suggestion
import React from 'react';

import clsx from 'clsx';

import { Switch, FormControlLabel, makeStyles } from '@material-ui/core';

```

import order is documented here: https://centreon.atlassian.net/wiki/spaces/RD/pages/141230093/Frontend

**Pull request merged**